notebooks/ch10/modeling_loss_functions.md

```diff
@@ -220,129 +303,148 @@ try_thetas(thetas=[12, 13, 14, 15, 16, 17],
            xlims=(11, 18))
 ```
 
-In the plots above, we've used integer $ \theta $ values in between 12 and 17. When we change $ \theta $, the cost seems to start high (at 10.92), decrease until $ \theta = 15 $, then increase again. We can see that the cost changes as $ \theta $ changes, so let's make a plot comparing the cost to $ \theta $ for each of the six $ \theta $s we've tried.
+
+![png](modeling_loss_functions_files/modeling_loss_functions_24_0.png)
+
+
+In the plots above, we've used integer $ \hat{\theta_0} $ values in between 12 and 17. When we change $ \hat{\theta_0} $, the loss seems to start high (at 10.92), decrease until $ \hat{\theta_0} = 15 $, then increase again. We can see that the loss changes as $ \hat{\theta_0} $ changes, so let's make a plot comparing the loss to $ \hat{\theta_0} $ for each of the six $ \hat{\theta_0} $s we've tried.
 
 
 ```python
 thetas = np.array([12, 13, 14, 15, 16, 17])
 y_vals = np.array([12.1, 12.8, 14.9, 16.3, 17.2])
-costs = [mse_cost(theta, y_vals) for theta in thetas]
+losses = [mse(theta, y_vals) for theta in thetas]
 
-plt.scatter(thetas, costs)
-plt.title(r'Cost vs. $ \theta $ when $ y = [ 12.1, 12.8, 14.9, 16.3, 17.2 ] $')
-plt.xlabel(r'$ \theta $ Values')
-plt.ylabel('Cost');
+plt.scatter(thetas, losses)
+plt.title(r'Loss vs. $ \hat{\theta_0} $ when $ y = [ 12.1, 12.8, 14.9, 16.3, 17.2 ] $')
+plt.xlabel(r'$ \hat{\theta_0} $ Values')
+plt.ylabel('Loss');
 ```
 
-The scatter plot shows the downward, then upward trend that we noticed before. We can try more values of $ \theta $ to see a complete curve that shows how the cost changes as $ \theta $ changes.
+
+![png](modeling_loss_functions_files/modeling_loss_functions_26_0.png)
+
+
+The scatter plot shows the downward, then upward trend that we noticed before. We can try more values of $ \hat{\theta_0} $ to see a complete curve that shows how the loss changes as $ \hat{\theta_0} $ changes.
 
 
 ```python
 thetas = np.arange(12, 17.1, 0.05)
 y_vals = np.array([12.1, 12.8, 14.9, 16.3, 17.2])
-costs = [mse_cost(theta, y_vals) for theta in thetas]
+losses = [mse(theta, y_vals) for theta in thetas]
 
-plt.plot(thetas, costs)
-plt.title(r'Cost vs. $ \theta $ when $ y = [ 12.1, 12.8, 14.9, 16.3, 17.2 ] $')
-plt.xlabel(r'$ \theta $ Values')
-plt.ylabel('Cost');
+plt.plot(thetas, losses)
+plt.title(r'Loss vs. $ \hat{\theta_0} $ when $ y = [ 12.1, 12.8, 14.9, 16.3, 17.2 ] $')
+plt.xlabel(r'$ \hat{\theta_0} $ Values')
+plt.ylabel('Loss');
 ```
 
-The plot above shows that in fact, $ \theta = 15$ was not the best choice; a $ \theta $ of around 14.7 would have gotten a lower cost! We can use calculus to find that minimizing value of $ \theta $ exactly. First, we start with our cost function:
+
+![png](modeling_loss_functions_files/modeling_loss_functions_28_0.png)
```

Comment:
Hmm.. I'm not sure if you have to re-run the code in the ipynb notebooks so that the images render properly :o

Reply:
Currently it's still theta in the image instead of theta_hat_0  but I noticed that the code has theta_hat_9